acceptance/catalog_services_test.go

```diff
@@ -19,6 +22,19 @@ var _ = Describe("Catalog Services", func() {
 			makeCatalogService(serviceName)
 		})
 
+		It("creates a catalog based service, with additional data", func() {
+			makeCatalogService(serviceName, `{ "db": { "name": "wordpress" }}`)
+			serviceInstanceName := fmt.Sprintf("service.org-%s.svc-%s", org, serviceName)
+			Eventually(func() string {
```

Comment:
Hmm, now that I think again, `Eventually` shouldn't be needed because `makeCatalogService` already waits for the service to appear. Also, if the service wasn't there the `Expect(err).ToNot(HaveOccurred())` would raise so the Eventually wouldn't help. I will try to remove the `Eventually` and test locally if it works.

Reply:
removed `Eventually` and it works fine.